homepage.css

```diff
@@ -31,7 +31,7 @@ ul{
 }
 
 li{
-	float:right;
+	float:left;
```

Comment:
15/08/2021: Make the link in the nav bar to be in the left side. To see how it go. (Could change it back to the left)